libraries/AP_Vehicle/AP_Vehicle.h

```diff
@@ -136,6 +136,22 @@ class AP_Vehicle : public AP_HAL::HAL::Callbacks {
     // false disables external leds)
     AP_Notify notify;
 
+protected:
+
+// Inertial Navigation EKF
+#if AP_AHRS_NAVEKF_AVAILABLE
+    NavEKF2 EKF2{&ahrs, rangefinder};
+    NavEKF3 EKF3{&ahrs, rangefinder};
+#if APM_BUILD_TYPE(APM_BUILD_ArduCopter) || APM_BUILD_TYPE(APM_BUILD_ArduSub)
+    // Copter and Sub force the use of EKF
+    AP_AHRS_NavEKF ahrs{EKF2, EKF3, AP_AHRS_NavEKF::FLAG_ALWAYS_USE_EKF};
```

Comment:
maybe move the flag set into the AP_AHRS_NavEKF.cpp constructor?

Reply:
Done.